lib/utils/dir-is-empty.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+const every = require('lodash/every');
+
+const debugLogRegex = /^ghost-cli-debug-.*\.log$/i;
+const importantDotfiles = ['.ghost-cli'];
+
+module.exports = function dirIsEmpty(dir) {
+    const files = fs.readdirSync(dir);
+
+    if (!files.length) {
+        return true;
+    }
+
+    return every(files, file => file.match(debugLogRegex) || (file.startsWith('.') && !importantDotfiles.includes(file)));
```

Comment:
```suggestion
    return files.filter(file => file.match(debugLogRegex) || (file.startsWith('.') && !importantDotfiles.includes(file));
```

Reply:
realized actually that node 6 implements Array.every natively, meaning we don't even need the lodash dep here.

Also, `every` is a better choice for the logic here imo - it returns a boolean consistently and will exit early if one file is found that does not match the condition.

Reply:
Whoops, brain fart! I was thinking we're returning a file list 🤦‍♂️

Reply:
done 😄